docs/HowTo/Withdrawal-Keys.md

```diff
@@ -0,0 +1,85 @@
+---
+title: How to update your withdrawal credentials
+---
+
+# Update your withdrawal credentials
+
+When you create a validator, it’s currently possible to set its [withdrawal](../Concepts/Withdrawals.md) address
+to a BLS address, or an Ethereum address. After the Capella fork, users must use an Ethereum address as their withdrawal
+address to allow automatics transfers.
+
+## Determine the withdrawal address type
+
+**Prerequisites**:
+
+- Access to the beacon node API endpoint. By default this is `localhost:5051`
+- Install [`curl`](https://curl.se/) and [`jq`](https://stedolan.github.io/jq/)
+
+The following shell script allows you to determine the withdrawal address of a given validator ID.
+
+=== "Script"
+
+    ```bash
+    VALIDATOR=<VALIDATOR_INDEX> \
+    curl http://localhost:5051/eth/v1/beacon/states/finalized/validators/$VALIDATOR | jq '.data | .validator.withdrawal_credentials'
+    ```
+
+=== "Example output"
+
+    ```
+    "0x00fc40352b0a186d83267fc1342ec5da49dbb78e1099a4bd8db16d2c0d223594"
+    ```
+
+In the script, specify the `<VALIDATOR_INDEX>` (for example `1`) that was provided when you joined the network. Alternatively
+you can specify the validator's public key.
+
+In the output, the first 4 characters of the string, in this case `0x00`, indicates the key is a BLS withdrawal
+key.
+
+## Update your withdrawal address
+
+### From a BLS withdrawal address to an Ethereum address
+
+!!! important
+
+    Teku does not offer functionality to update your withdrawal key from a BLS withdrawal address to
+    an Ethereum address. Use a tool like [`ethdo`](https://github.com/wealdtech/ethdo/blob/master/docs/changingwithdrawalcredentials.md)
+    instead.
```

Comment:
I might have another go at this...
```
Teku does not offer functionality to create a signed withdrawal credential change.
A tool such as `ethdo` can allow you to generate this signed message, which can be submitted directly to your beacon-node if your rest-api is active.
```

I don't know if we mention as an aside that one reason for not allowing this is that typically it is not a good design choice to have both your withdrawal keys and validator keys available at the same place at the same time?

Reply:
Raised a separate issue to address this: https://github.com/ConsenSys/doc.teku/issues/427